### DISCUSSION TITLE
Is that possible to make the entire card clickable ? Thank you.

### DISCUSSION THREAD

**Discussion by creditket** [hbstack/site#759] [General]
Is that possible to make the entire card clickable ? Thank you.
<img width="984" alt="Untitled" src="https://github.com/hbstack/site/assets/93809455/f0727048-3502-4763-b8bd-0e3e88b2c6b0">

**Comment by razonyang** [comment]
No, it's invalid that nest `<a>` inside a `<a>` elements.

**Reply by creditket** [reply]
ok, then can we just make the element/the card clickable only?  assume that we just don't want any link/<a> inside the card?  (since it make more sense to match more the card theme)

**Reply by razonyang** [reply]
You can create [layouts/partials/hb/modules/blog/post/card.html](https://github.com/hbstack/blog/blob/main/layouts/partials/hb/modules/blog/post/card.html) and tweak the template to suit your needs.

> Please note that you may not got fully updates when updating modules in this way, you customized templates are always prioritiest.

You can override any templates you want, you can search on GitHub via classname to locate the target template, for example, https://github.com/search?q=org%3Ahbstack%20hb-blog-post-card%20card&type=code
